test.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title> FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <section id="welcome-section">
+        <h1>Hey am Jumite</h1>
+        <p>a web developer and designer</p>
+
+      </section>
+      <section id="projects">
+          <h2 class="projects-section-header">These are 
+              some of my projects
+          </h2>
+          <div class="project-tile">
+          </div>
+          
+          <a href="#"></a>
+      </section>
+
+      <nav id="navbar">
+        <a href="#"></a>
+      </nav>
+    <a href="profile-link"></a>
```

Comment:
- [ ]  Ensure the correct position, to ensure your code looks organized.

Reply:
It is good practice when giving a review to be polite when addressing the reviewer. You could prase this comment like, "Proper code indentation makes your code more readable and easier to understand by other developers looking to contribute to your codebase, I would advise you to follow the industry standards for code [standards for code indentation".](https://en.wikipedia.org/wiki/Indentation_style)